File: 30_days_of_coding/Day6_Review.cs
class Solution {
    static void Main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution */
        int n = Convert.ToInt32(Console.ReadLine());
        string odd;
        string even;
        
        for(int i = 0; i < n; i++){
            string word = Console.ReadLine();
            odd = null;
            even = null;
            for(int j = 0; j < word.Length; j++){
                if(j % 2 == 0){
                    even += word[j];
                }else{
                    odd += word[j];
                }
            }
            Console.WriteLine(even + " " + odd);
        }
    }
}

File: 30DaysOfCodingJAVA/Day6_Review.java
public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        
        Scanner scan = new Scanner(System.in);
        int n = Integer.parseInt(scan.nextLine());
        for(int i = 0; i < n; i++){
            StringBuilder even = new StringBuilder();
            StringBuilder odd = new StringBuilder();
            String temp = scan.nextLine();
            for(int j = 0; j < temp.length(); j++){
                if(j % 2 == 0){
                    even.append(temp.charAt(j));
                } else{
                    odd.append(temp.charAt(j));
                }
            }
            System.out.println(even + " " + odd);
        }
        scan.close();
    }
}

//String temp = scan.nextLine();
//            for(int j = 0; j < temp.length(); j++){
//                if(j % 2 == 0){
//                    even += temp.substring(j, j + 1);
//                } else{
//                    odd += temp.substring(j, j + 1);
//                }
//            }

// char[] temp = scan.nextLine().toCharArray();
//            for(int j = 0; j < temp.length; j++){
//                if(j % 2 == 0){
//                    even += temp[j];
//                } else{
//                    odd += temp[j];
//                }
//            }
